src/mame/drivers/firebeat.cpp

```diff
@@ -431,6 +471,12 @@ void firebeat_state::machine_start()
 	m_maincpu->ppcdrc_add_fastram(0x00000000, 0x01ffffff, false, m_work_ram);
 }
 
+void firebeat_state::device_resolve_objects()
+{
+	printf("Called firebeat_state::device_resolve_objects\n");
```

Comment:
Remember to remove this printf if you haven’t already.

Reply:
No worries, already taken care of in https://github.com/mamedev/mame/pull/7721/commits/8382b21536e90f61fce720b6e004eb77691c9d75 when I noticed the message after running it again.